app/views/endemics/you-can-claim-multiple.njk

```diff
@@ -0,0 +1,54 @@
+{% extends './layouts/layout.njk' %}
+
+{% set pageHeadingText='You can claim for more than one species as part of this agreement' %}
+
+{% block pageTitle %}{{ pageHeadingText }} - {{ serviceName }} - GOV.UK{% endblock %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: "Back",
+    href: backLink,
+	  attributes: {id: "back"}
+  }) }}
+  {{ nameSbiTag({name: organisation.name, sbi: organisation.sbi}) }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">{{ pageHeadingText }}</h1>
+      <p class="govuk-body">Under this agreement, you can claim funding for the following species:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>beef cattle</li>
+        <li>dairy cattle</li>
+        <li>pigs</li>
+        <li>sheep</li>
+      </ul>
+      <p class="govuk-body">You can do a maximum of 3 reviews and 3 follow-ups per species. You must do all your reviews and follow-ups by 19 June 2027.</p>
+      <p class="govuk-body">If you did a review in the old annual health and welfare service, it will not count towards your limit.</p>
+      <p class="govuk-body">You need to make a separate claim for each species.</p>
+      <br>
+
+      <form method="POST" autocomplete="off" novalidate="novalidate" id ="submitYouCanClaimMultipleForm">
+        <input type="hidden" name="crumb" value="{{crumb}}"/>
+        <div class="govuk-button-group">
+          {{ govukButton({
+          attributes: { "aria-label": agreementStatus[0].text + " - " + pageHeadingText },
```

Comment:
Do we need aria labels when the button has text already? In this case it's going to read out the same text as the button has along with the entire page heading for each button.

Reply:
I did by example on this, copied from Reviews page, will have a think can get back to you